internal/actions/sync_branch.go

```diff
@@ -43,14 +42,10 @@ type SyncBranchResult struct {
 
 type (
 	SyncBranchContinuation struct {
```

Comment:
Maybe makes sense to just rename this to `SyncBranchState` at this point?

Reply:
I think continuation makes sense. It's indeed the continuation post rebase.